config/config_test.go

```diff
@@ -95,6 +95,8 @@ func TestConfig_BuildBlockAdapter(t *testing.T) {
 			t.Fatalf("expected an s3 block adapter, got something else instead")
 		}
 	})
+
+	// TODO(barak): add gs block adapter test
```

Comment:
TODO

Reply:
will look for a way to initialize google storage client without credentials or remove the TODO comment - thanks.

Reply:
done